tests/crypto/test_cases/test_vectors_hmac.c

```diff
@@ -0,0 +1,346 @@
+/*$$$LICENCE_NORDIC_STANDARD<2019>$$$*/
```

Comment:
Is this right? Or do we need an SPDX line? I don't see this style used very often in NCS

Reply:
You are right, changed in edd6a4a